package.json

```diff
@@ -2,10 +2,26 @@
   "name": "root",
   "private": true,
   "scripts": {
-    "start": "lerna run start --parallel"
+    "start": "lerna run start --parallel",
+    "prepare": "husky"
+  },
+  "lint-staged": {
+    "**/*.{js,jsx,ts,tsx}": [
+      "eslint --fix",
+      "prettier --write"
+    ]
   },
   "dependencies": {
     "lerna": "^8.1.8",
     "vite": "^5.4.8"
```

Comment:
Also, move these dependencies in devdepencies. I don't think they are required in dependencies.

Reply:
okay